main.cpp

```diff
@@ -51,42 +51,216 @@ struct CarWash            //1) a U.D.T. with a random number of member variables
 /*
  1)
  */
+struct Person
+{
+    int numHands = 2;
+    float height = 1.68f; // in [m]
+    
+    struct Hand
+    {
+        bool isLeftHand = false;
+        float fingerLength = 0.082f;        //2) in [m]
+    };
+    
+    void isMale( Person person );
+    
+    Hand myLeftHand;
+};
```

Comment:
The next project part for this uses the `Person` struct from the video.  you'll need to change this UDT's type name to something else.  Maybe `Human` would work? 

- [ ] change UDT name